tests/unit_tests/test_pychop.py

```diff
@@ -252,7 +280,12 @@ def test_chop_times(matrix, pychop_nonfermi_data):
 
     for aa, ee in zip(actual, expected):
         for a, e in zip(aa, ee):
-            assert_allclose(a, e, rtol=0, atol=1e-8)
+            try:
+                assert_allclose(a, e, rtol=0, atol=1e-8)
+            except AssertionError:
+                print(actual)
+                print(expected)
```

Comment:
This looks like debugging code. I'm surprised the `assert_allclose` isn't giving useful output?

Reply:
Since this is in a loop, `assert_allclose` prints out  only the 2-element 1D innermost array, which is enough for comparison, but doesn't tell us anything about which loop the error occured on or what the other values in the list of lists are, hence the prints of the entire thing. 

And, the `assert_allclose` can't be called on the entire list of lists either because it is often irregular, so it can't be packed into a nice NxMxK array.

Reply:
I see! That makes sense.